sdk/core/azure-core/src/main/java/com/azure/core/util/implementation/ReferenceManagerImpl.java

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.implementation;
```

Comment:
I'm not a fan of multiple implementation packages - I'd rather have a com.azure.core.implementation.*.*.* style than this.

Reply:
Was just adding a missing `package-info` in this case, we should discuss cleaning this up more in the future as we also have other package paths to implementation in Core, ex `com.azure.core.implementation`.